mms/model_service_worker.py

```diff
@@ -153,13 +157,14 @@ def retrieve_model_input(self, model_inputs):
         :return:
         """
 
-        model_in = []
-        for ip in model_inputs:
+        model_in = dict()
```

Comment:
This should be a list, not map, input can have duplicate name.

Reply:
If inputs have duplicate names, it can't be used by the model. How would the model know which is the first input and which one is the second input?